config/locales/en_LK.yml

```diff
@@ -0,0 +1,40 @@
+en_LK:
+  facility_group: "district"
+  facility_group_zone: "Zone (TBD)"
+  helpers:
+    label:
+      facility:
+        street_address: "House number and street"
+        village_or_colony: "Town/Area"
+        zone: "Zone (TBD)"
+        district: "District *"
+        state: "Province *"
+        country: "Country *"
+        pin: "Postal Code"
+      facility_group:
+        state: "Province"
+      address:
+        street_address: "House and street number"
+        village_or_colony: "Town/Area"
+        zone: "Zone (TBD)"
+        district: "District"
+        state: "Province *"
+        country: "Country *"
+        pin: "Postal Code"
+      drug:
+        frequency:
+          one_per_day: "Daily"
+          two_per_day: "BD (morning and night)"
```

Comment:
Is this the copy shown in the app? Can we check if this length causes display/layout issues?

Reply:
This copy is not being shown anywhere on the app. We will have to keep it in mind when we pick up https://app.clubhouse.io/simpledotorg/story/2915/implement-drugfrequencyfactory-drugfrequencyprovider-for-providing-drug-frequencies-label-depending-on-the-country.